tests/test_time.py

```diff
@@ -1358,8 +1358,7 @@ def test_UTCGPS(self):
         """testing get UTC from GPS"""
         t1 = t.Ticktock([6.93882013e+08, 6.93964813e+08], 'GPS')
         expected = t.Ticktock(['2002-01-01T01:00:00', '2002-01-02'])
-        #        numpy.testing.assert_array_equal(t1, expected)
-        self.assertTrue((t1 == expected).all())
+        numpy.testing.assert_array_equal(t1, expected)
```

Comment:
This likely needs to be ` np.testing.assert_array_almost_equal` as these are possibly floats.

Reply:
They're float type, but integral valued, and thus the calculation should carry through exactly with no loss of precision. If that ever didn't happen I would want the test to fail.